### DISCUSSION TITLE
How can I schedule tweets natively?

### DISCUSSION THREAD

**Discussion by sevenleo** [tweepy/tweepy#1525] [Questions] [answered]
Can I use Tweepy API to schedule posts?

![image](https://user-images.githubusercontent.com/5843634/107843592-3ae73c00-6dab-11eb-8ea9-c131cdf4763b.png)

**Comment by Harmon758** [comment, marked as answer]
This is only possible with the [Twitter Ads API](https://developer.twitter.com/en/docs/twitter-ads-api) (see https://developer.twitter.com/en/docs/twitter-ads-api/creatives/guides/scheduled-tweets-guide), which Tweepy does not support.

**Comment by AustinGitHub** [comment]
Is there any plans to support for this? This would be very useful

**Reply by Harmon758** [reply]
This is only possible with the [Twitter Ads API](https://developer.twitter.com/en/docs/twitter-ads-api), which there are no plans to support.
